crates/rpc/rpc/src/eth/core.rs

```diff
@@ -151,7 +153,7 @@ where
     type Pool = Pool;
     type Evm = EvmConfig;
     type Network = Network;
-    type PayloadBuilder = ();
+    type PayloadBuilder = EthereumPayloadBuilder<EvmConfig>;
```

Comment:
we need to solve this a little bit differently, 
similar to how provider is integrated, via a generic. but for this pr this  should be okay.